### ISSUE TITLE
FAIRmetrics score on the profile of each published dataset?

### ISSUE THREAD

**Issue by xgmachina** [FAIR-EG/consultation#6]
[//]: # "==Do not write above this line==
Instructions for posting issues: 
(1) Review what is already there. Perhaps a comment to an existing issue would be more appropriate than opening a new one?
(2) Write your post below using Markdown (as per https://guides.github.com/features/mastering-markdown/ ) or just plain text.
(3) Don't worry about these introductory lines - you can leave or delete them, as they won't display anyway (you can check this via Preview).
(4) Hit the 'Submit new issue' button.
==Write below this line=="
I have a naive idea about a rose diagram of FAIRmetrics on the profile of each published dataset. The idea is derived from the Altermetrics that is widely used for scientific publications now.

A rose diagram with a overall data quality value can be very transparent and attractive on the profile of a dataset. The challenge, as already mentioned by a few colleagues, is methods to do the quantitative estimation of the four elements in FAIR. Altermetrics uses datasets from social media to estimate the impact of a publication. For the elements of findability, accessibility, interoperability and reusability, there could be some automated ways to get records if the dataset has a unique identifier such as DOI or URI. Still, we need to carefully design a model and a mechanism to evaluate each item in FAIR, and then generate a overall score.

**Comment by band** [comment]
Marshall, this is a good idea. I think there is a straightforward way to define measurable properties for each principle, treating each as a system requirement. The work will be in agreeing on measurements that are practicable for both data contributions and data repository operations. Much of the high-level context conversation re FAIR-ness seems to be settled (for now).

See my comment on Issue 5: https://github.com/FAIR-Data-EG/consultation/issues

**Comment by Daniel-Mietchen** [comment]
Thanks &mdash; needs sync with #5.

**Comment by xgmachina** [comment]
@band Yes, the issue I raised here is a concise and visualized output of the properties you mentioned in #5. The guidelines under each FAIR item offered some clues to an initial prototype. https://www.force11.org/group/fairgroup/fairprinciples

**Comment by xgmachina** [comment]
@Daniel-Mietchen perhaps 'visualization' can be added as a label.

**Comment by sjDCC** [comment]
I've added visualisation as a label.

It's also worth looking at the DANS work on assessing the FAIRness of datasets. They've proposed some ways to show this on metadata records. See for example [this recent presentation](https://planeurope.files.wordpress.com/2015/03/doorn-fair-interoperability-poznan-plan-e-april-2017.pdf) and [blog post](http://blog.ukdataservice.ac.uk/fair-data-assessment-tool)

**Comment by dr-shorthair** [comment]
Also see https://confluence.csiro.au/display/OZNOME/Rating+systems+and+maturity+models+for+data+publication+and+sharing#Ratingsystemsandmaturitymodelsfordatapublicationandsharing-OFLWtestcases
